### PR TITLE
use class instead data-hook for gateway partial

### PR DESCRIPTION
**Pull request by ccarruitero**
Given that other fields in the partial have a data-hook and class to allow easy customization, I think will be good that `card name` field have the same

### CODE REVIEW

File: frontend/app/views/spree/checkout/payment/_gateway.html.erb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <%= image_tag 'credit_cards/credit_card.gif', id: 'credit-card-image' %>
 <% param_prefix = "payment_source[#{payment_method.id}]" %>
 
-<p class="field">
+<p class="field" data-hook="card_name">
```
Comment (kennyadsl) [inline frontend/app/views/spree/checkout/payment/_gateway.html.erb]: I think the general feeling is to remove `data-hook` attributes. To follow this path you could use a class instead, like:

```
<p class="field payment-form-card-name">
```

and add a similar class to the other elements for consistency (number and expiration).

You should be able to hook this element with a class as well, right?

Comment (ccarruitero) [reply frontend/app/views/spree/checkout/payment/_gateway.html.erb]: Yeah, there shouldn't  be a problem.

I'm going to update the elements to use class.

**Comment by kennyadsl** [comment]
@ccarruitero Can you please rebase this?

**Comment by kennyadsl** [comment]
@ccarruitero ping 🙂

**Comment by aitbw** [comment]
Closing now that #3182 was merged, thanks @ccarruitero!

**Pull request closed without merging**